test/library/cluster_operator.go

```diff
@@ -3,38 +3,50 @@ package library
 import (
 	"fmt"
 	"testing"
+	"time"
 
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/wait"
 
 	configv1 "github.com/openshift/api/config/v1"
-	configclient "github.com/openshift/client-go/config/clientset/versioned/typed/config/v1"
+	configv1client "github.com/openshift/client-go/config/clientset/versioned/typed/config/v1"
 	clusteroperatorhelpers "github.com/openshift/library-go/pkg/config/clusteroperator/v1helpers"
 )
 
 // WaitForKubeAPIServerClusterOperatorAvailableNotProgressingNotDegraded waits for ClusterOperator/kube-apiserver to report
 // status as active, not progressing, and not failing.
-func WaitForKubeAPIServerClusterOperatorAvailableNotProgressingNotDegraded(t *testing.T, client configclient.ConfigV1Interface) {
-	err := wait.Poll(WaitPollInterval, WaitPollTimeout, func() (bool, error) {
+func WaitForKubeAPIServerClusterOperatorAvailableNotProgressingNotDegraded(t testing.TB, client configv1client.ConfigV1Interface) {
+	t.Helper()
+	t.Log("Waiting for ClusterOperator/kube-apiserver to be Available: true, Progressing: false, Degraded: false, Done: false")
+	time.Sleep(time.Minute) // make sure we are not racing against an initial observation of change
```

Comment:
suspiciuos. Why this?

Reply:
I can remove it, Mo added it.

Reply:
he might have added it because we don't know when encryption has started and it might happen that the condition will be satisfied immediately when encryption hasn't started.

Reply:
we could try to observe conditions from individual encryption controllers instead of looking at the aggregate condition but it is not that bad.

Reply:
we could also add a condition that would tell us that migration is done (all `GRs` migrated) and watch it.